net/wol/Makefile

```diff
@@ -1,6 +1,6 @@
 PLUGIN_NAME=		wol
-PLUGIN_VERSION=		1.0
-PLUGIN_REVISION=	1
+PLUGIN_VERSION=		2.0
+PLUGIN_REVISION=	0
 PLUGIN_DEPENDS=		wol
 PLUGIN_COMMENT=		Wake on LAN Service
 PLUGIN_MAINTAINER=	franco@opnsense.org
```

Comment:
you should take this if you rewrite it :)

Reply:
It's yours ;)